### ISSUE TITLE
Error when selecting item in Itinerary

### ISSUE THREAD

**Issue by HilaryN** [cyclestreets/android#516]
Often when I try and select an item in the Itinerary, the app crashes and the logcat shows similar message to here:

https://stackoverflow.com/questions/54047971/android-8-1-9-0-removeviewview-is-not-supported-in-adapterview

**Comment by jezhiggins** [comment]
Yes, confirmed. I got it to trigger just by switching between the map and itinerary view the hamburger menu, so it's probably related to the change between the two views rather specifically the item select.
